src/containers/shared/components/Transaction/PaymentChannelClaim/parser.ts

```diff
@@ -1,18 +1,11 @@
-import type { Meta } from '../../../types'
-import { PaymentChannelClaim, PaymentChannelClaimInstructions } from './types'
+import type { PaymentChannelClaim, TransactionMetadata } from 'xrpl'
+import { PaymentChannelClaimInstructions } from './types'
 import { formatAmount } from '../../../../../rippled/lib/txSummary/formatAmount'
+import { findNode } from '../../../transactionUtils'
 
 const hasRenew = (flags: number): boolean => !!(0x00010000 & flags)
 const hasClose = (flags: number) => !!(0x00020000 & flags)
```

Comment:
Aren't the flags all constants somewhere? That'd make this code more readable.

Reply:
The flags in the explorer are stored in a map that are reversed, flag value to flag name.  Will have to redo that to clean this up.